WordPress/src/main/java/org/wordpress/android/ui/compose/components/menu/dropdown/MenuElementData.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui.compose.components.menu.dropdown
+
+import androidx.annotation.DrawableRes
+
+sealed class MenuElementData(
+    open val text: String,
+    @DrawableRes open val leadingIcon: Int,
```

Comment:
❓  Looking at the Composables implementations it feels like if this was `nullable` it would make some of the logic easier, is there any benefit of having a non-null `leadingIcon` with the `NO_ICON` default?

Reply:
This was pretty much personal preference. I usually prefer having default values over having a nullable field to avoid having to unwrap it all the time (e.g. with a `.let`). 

If you mean this part, for example:
```
if (element.leadingIcon != NO_ICON) {
            {
                Icon(
                    painter = painterResource(id = element.leadingIcon),
                    contentDescription = null,
                )
            }
        } else null
```

I believe we'll still need something like:
```
leadingIcon = element.leadingIcon?.let {
            {
                Icon(
                    painter = painterResource(id = element.leadingIcon),
                    contentDescription = null,
                )
            }
        } ?: null
```

This happens because if the `leadingIcon` is not set we must pass `null` to the `DropdownMenuItem`, otherwise there's an empty space before the text, as if there was a hidden icon there. I believe the only way might be if we had the actual Compose `Icon` as a property of `MenuElementData`, but it wouldn't be testable AFAIU.
Please let me know if you this is what you meant.